distribution/docs/src/main/resources/content/_quickstart/quickstart-installing.adoc

```diff
@@ -186,10 +186,13 @@ ${branding-lowercase}${at-symbol}local>
 
 If ${branding} is being installed on a remote server that has no user interface some additional steps must be taken prior to starting the system.
 
-. Update any references to localhost in the following files. These references to localhost should be updated to match either the hostname or IP of the system.
-** `${home_directory}/etc/custom.system.properties`
-** `${home_directory}/etc/users.properties`
-** `${home_directory}/etc/users.attributes`
+. Update the `${home_directory}/etc/custom.system.properties` file.
```

Comment:
✏️ This is a nit/style preference. There is no need to have bullets when there is only one item in the list. I prefer something like this instead. 

If ${branding} is being installed on a remote server that has no user interface some additional steps must be taken prior to starting the system.

```
1. Update the ${home_directory}/etc/custom.system.properties file. The entry org.codice.ddf.system.hostname=localhost should be updated to match either the hostname or IP of the system.
2. Update the ${home_directory}/etc/users.properties file. Change the localhost=localhost[…​] entry to <HOSTNAME>=<HOSTNAME>, or <IP>=<IP>. (Keep the rest of the line as is.)
3. Update the ${home_directory}/etc/users.attributes file. Change the "localhost" entry to "<HOSTNAME>" or "<IP>".
4. From the console go to ${home_directory}/etc/certs.
  a. If using a hostname run: sh CertNew.sh -cn <hostname> -san "DNS:<hostname>" (or CertNew -cn <hostname> -san "DNS:<hostname>" on windows).
  b. If using an IP address run: sh CertNew.sh -cn <IP> -san "IP:<IP>" (or CertNew -cn <IP> -san "IP:<IP>" on windows).
5. Proceed with starting the system and continue as usual.
```

Reply:
✏️ These 5 directions might be simpler to split into 2 seperate set of directions: one for hostname and one for IP address.

Reply:
That's a good point. I considered adding a note that you should be consistent between methods, but I think the separation will clear up any ambiguity

Reply:
![Screen Shot 2019-05-01 at 2 29 36 PM](https://user-images.githubusercontent.com/10619999/57043923-c9fa8280-6c1d-11e9-872f-3a328a45d0b3.png)